components/app.js

```diff
@@ -10,8 +11,23 @@ class App {
         console.error(error);
     }
 
+    //gets grades from getGrades
+    //updates the grades in the table (from constructor param)
+    //computes the grade average and passes it to this.pageHeader.updateAverage
     handleGetGradesSuccess(grades) {
         this.gradeTable.updateGrades(grades);
+        
+        let classSize = 0;
+        let gradeSum = 0;
+
+        grades.forEach(student => {
+            classSize++;
+            gradeSum += student.grade;
+        })
+
+        const average = gradeSum / classSize;
```

Comment:
Not a big deal but instead of counting the grades with `classSize` you could of just used `grades.length` 😃